src/api.ts

```diff
@@ -142,6 +142,12 @@ export default class Chromeless<T extends any> implements Promise<T> {
     return this
   }
 
+  setDocumentContent(html: string): Chromeless<T> {
```

Comment:
I'm a bit against naming public methods the same thing as their counterparts in CDP. It (in my head at least) feels like it "locks" us a bit to the implementation details of CPD as opposed to just offering a feature regardless of how it's implemented.

I'm not too strongly tied to this opinion, but wanted to gather folks' thoughts